tracer/build/_build/Build.Steps.cs

```diff
@@ -2212,30 +2228,31 @@ private record ParsedLogLine(DateTimeOffset Timestamp, LogLevel Level, string Me
         TargetFramework framework = null,
         bool noRestore = true,
         bool noDependencies = true,
-        bool setNuget = false,
-        bool setPackages = false
-        )
+        (string, object)[] additionalProperties = null)
     {
-        DotnetBuild(new [] { project.Path }, platform, framework, noRestore, noDependencies); 
+        DotnetBuild(new [] { project.Path }, platform, framework, noRestore, noDependencies, additionalProperties); 
     }
 
     private void DotnetBuild(
         IEnumerable<AbsolutePath> projPaths,
         MSBuildTargetPlatform platform = null,
         TargetFramework framework = null,
         bool noRestore = true,
-        bool noDependencies = true
-        )
+        bool noDependencies = true,
+        (string, object)[] additionalProperties = null)
```

Comment:
oh no, this feels like the point where `DotnetBuild` has outlived its usefulness and becomes too hard to understand 😅 I'd be strongly inclined to _not_ add this, and to just manually call the effective `DotNetBuild` for the debugger targets.

Reply:
I've removed `additionalProperties` here d92c66f92a736ae21a4fcd11b464fdaaa34539e5